tests/foreman/api/test_repository.py

```diff
@@ -1387,6 +1387,48 @@ def test_positive_sync_rh_app_stream(self):
         """
         pass
 
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    @pytest.mark.destructive
+    @pytest.mark.run_in_one_thread
```

Comment:
```suggestion
```
You don't need to make this destructive since it isn't rebooting the system. The same is true for the other tests in this PR that don't involve reboots/manipulating services.